web/src/services/theme/reearthTheme/types.ts

```diff
@@ -5,85 +5,46 @@ export type TempTheme = Theme & {
   classic: ClassicTheme;
 };
 
-type InteractiveElementTheme = {
-  main: string;
-  hover: string;
-  disable: string;
-  content: string;
-  contentHover?: string;
-  contentDisable?: string;
-};
-
-// Note: anything typed unknown is not set yet and just anticipated with high likelihood
-// But, might not be necessary so keep in mind. 2023/06/15 @KaWaite
-
 export type Theme = Common & {
-  general: {
-    select: string;
-    bg: {
-      transparent: string;
-      veryWeak: string;
-      weak: string;
-      main: string;
-      strong: string;
-      veryStrong: string;
-    }; // Do we need????
-    border: string;
-    button: {
-      primary: InteractiveElementTheme;
-      secondary: InteractiveElementTheme;
-      danger: InteractiveElementTheme;
-    };
-    content: {
-      weak: string;
-      main: string;
-      strong: string;
-    };
+  bg: {
+    transparentBlack: string;
+    0: string;
+    1: string;
+    2: string;
+    3: string;
+    4: string;
+  };
+  select: {
+    light: string;
+    main: string;
+    strong: string;
+  };
+  item: {
+    default: string;
+    hover: string;
+  };
+  outline: {
+    weakest: string;
+    weak: string;
+    main: string;
+  };
+  primary: {
+    main: string;
+    weak: string;
   };
-  dashboard: {
-    workspace: unknown;
-    quickStart: unknown;
-    projectList: unknown;
+  text: {
```

Comment:
Also, can you talk with red about updating the figma theme to use `content` here instead of `text`, since this value is used for text, icons, etc, which `content` is more meaningful I think

Reply:
done